web/viewer.css

```diff
@@ -690,6 +707,9 @@ body {
   display: inline-block;
 }
 .splitToolbarButton > .toolbarButton {
+  /*#if !MOZCENTRAL*/
+  float: var(--float-inline-start);
+  /*#endif*/
```

Comment:
Shouldn't we wrap this in an if/else/endif instead so that we don't insert double `float` statements for non-Firefox targets? I'm not sure what would happen then since the second one is unsupported and might break.

Reply:
I was trying to implement this patch such that we'd make no changes to the *built* Firefox viewer, however your comment highlights the fact that this clearly ended up hurting readability and maintainability of this code. Please check if the updated patch is better.

Reply:
Yes, this seems better to me. Thanks!